doc/source/tune.rst

```diff
@@ -1,297 +1,98 @@
-Ray Tune: Hyperparameter Optimization Framework
-===============================================
+Tune: Scalable Hyperparameter Search
```

Comment:
I think the previous page was more informative. The current one seems to be lighter info-wise, and is partially redundant with the user guide. Perhaps they can be merged together?

Reply:
I was hoping to make the landing page lighter.

This is mainly following the formats of these library landing pages:

https://keras.io/
http://docs.python-requests.org/en/master/, https://github.com/requests/requests
https://www.nltk.org/